docs/sources/administration/provisioning.md

```diff
@@ -30,33 +30,19 @@ Checkout the [configuration](/installation/configuration) page for more informat
 
 ### Using Environment Variables
 
-All options in the configuration file (listed below) can be overridden
-using environment variables using the syntax:
+It is possible to use environment variable interpolation in all 3 provisioning config types. Allowed syntax
```

Comment:
@bergquist Added the documentation here. I removed the old text because it was talking about using env vars for main configuration.ini and was duplicate from installation/configuration.md so it seemed out of place and confusing here.